packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -834,6 +834,9 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     assert(() { _debugLocked = false; return true; }());
   }
 
+  /// The top-most [Route] in this navigator
```

Comment:
one trivial nit: the documentation should end with a period.

Reply:
Done.